### PR TITLE
Zee test

### PR DESCRIPTION
**Pull request by Zee314159**

### CODE REVIEW

File: scripts/test.sh [hunk 1]
```diff
@@ -180,4 +181,10 @@ if [[ $TEST_WITH_TESTSUITE = 1 ]]; then
     fi
 fi
 
+#Make docs
+if [[ $TEST_WITH_DOCS = 1 ]]; then
+   echo ">> Running make docs"
+   make docs
```
Comment (lbonn) [inline scripts/test.sh]: The `if [[ $TEST_DRYRUN != 1 ]]; then` check is missing

Comment (patrickvacek) [reply scripts/test.sh]: For consistency with the other cases, it would be good to use `run_make`, `add_failure`, and setting `+x`/`-x`.

Comment (Zee314159) [reply scripts/test.sh]: Sure.That makes sense.

**Comment by lbonn** [comment]
Cool! Could you rebase everything to one or two commits?

File: scripts/test.sh (hunk 1 above)
Comment (Zee314159) [reply scripts/test.sh]: SORRY...so many unintended commits...still on my way to getting along with git. I'll practice somewhere else first next time, and I'll clean this mess up later.

Comment (patrickvacek) [reply scripts/test.sh]: Sounds fine, let us know if you need help!

Comment (Zee314159) [reply scripts/test.sh]: ok, I'll try to figure it out by myself first. If it takes too much time unnecessarily, I'll just stop and turn to you for help. Thank you for your patience!

**Comment by patrickvacek** [comment]
@zabbal What is the connection to this PR?

**Comment by zabbal** [comment]
> @zabbal What is the connection to this PR?

Seemed relevant because googletest version is bumped in one of the commits and the ticket is about handling it as a dependency.

**Comment by patrickvacek** [comment]
> Seemed relevant because googletest version is bumped in one of the commits

Strange, we did bump the version relatively recently, but in a different, unrelated PR. I think we can safely move the conversation to the Jira ticket!

**Pull request closed without merging**